test/module/irohad/peerservice/peerservice_test.cpp

```diff
@@ -33,7 +33,7 @@ class Service : public testing::Test {
   Service() {
     std::vector<ed25519::keypair_t> kps(NPEERS);
     std::generate_n(kps.begin(), NPEERS,
-                    []() { return create_keypair(create_seed()); });
+                    []() { return create_keypair(); });
```

Comment:
can be just `std::generate_n(kps.begin(), NPEERS, create_keypair)`
and btw `std::generate(kps.begin(), kps.end(), create_keypair)` seems less error-prone for me